api/src/main/java/io/opencensus/metrics/export/Metric.java

```diff
@@ -48,7 +50,8 @@
    */
   public static Metric create(MetricDescriptor metricDescriptor, List<TimeSeries> timeSeriesList) {
     checkTypeMatch(metricDescriptor.getType(), timeSeriesList);
```

Comment:
Does it make sense do ```checkNotNull``` and ```checkListElementNotNull``` on timeSeriesList?

Reply:
Make sense, updated. Also added a few tests on that.